apps/expo/src/components/muscleGroup/GenderSwitch.test.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { render } from '@testing-library/react-native'
+import Toggle from '~/components/toggle/toggle'
```

Comment:
remove unused import

Reply:
also, move this file under __tests__ folder

Reply:
Moved and removed